### PR TITLE
ops: Celery fix

### PR DESCRIPTION
**Pull request by jannable**

### CODE REVIEW

File: django/initial_setup.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+echo "Installing Python dependencies..."
+pip install -r requirements.txt || { echo "Error: Pip install failed"; exit 1; }
```
Comment (haitmo) [inline django/initial_setup.sh:4]: This is taken care of in the docker file already no?

File: setup/.env.example [hunk 2]
```diff
@@ -1,31 +1,87 @@
...
-CRITICALITY=NonEssential # Criticality of the application (e.g. Essential, NonEssential)
-OWNER="Business Analytics Centre" # Owner of the application
-
-# Application
-DJANGO_ENV=DEV # Django environment (e.g. DEV, UAT, PROD)
-DJANGO_DEBUG=True # Django debug mode (e.g. True, False)
-
-# OpenAI Usage Capacity
+# Version number for this environment file
+ENV_VERSION=1.2
```
Comment (haitmo) [inline setup/.env.example:2]: Shouldn't this be 1.1?

File: setup/.env.example [hunk 3]
```diff
@@ -1,31 +1,87 @@
...
+### APPLICATION URL ###
+
+# Option 1: Custom Domain (Production)
+# Uncomment and set this for environments with a custom domain
+# SITE_URL=https://otto.justice.gc.ca
+
+# Option 2: Azure-provided Domain
+# Uncomment and set this for environments without a custom domain
+# Format: otto-<environment>-<uniqueidentifier>
+DNS_LABEL=otto-sandbox-abc123
```
Comment (haitmo) [inline setup/.env.example:29]: What is the uniqueidentifier for?

File: setup/.env.example [hunk 4]
```diff
@@ -1,31 +1,87 @@
...
+### APPLICATION VARIABLES ###
+
+# Django environment (e.g. DEV, UAT, PROD)
+DJANGO_ENV=DEV 
+
+# Django debug mode (e.g. True, False)
+DJANGO_DEBUG=True
+
+# Application admin (comma-separated list of UPNs)
+OTTO_ADMIN=jason.annable@justice.gc.ca,hamza.aitmohamed@justice.gc.ca,jules.kuehn@justice.gc.ca
```
Comment (haitmo) [inline setup/.env.example:67]: Can we leave off email addresses from the repo and provide an example of an email address?

**Review by haitmo — DISMISSED** [review]

File: django/initial_setup.sh (hunk 1 above)
Comment (jannable) [reply django/initial_setup.sh:4]: Oops, meant to remove it earlier.

File: setup/.env.example (hunk 3 above)
Comment (jannable) [reply setup/.env.example:29]: This is a public repo and it's conceivable someone else might try to stand this up in the same canadacentral region. the unique identifier is to cover the theoretical possibility of a conflict.

**Review by haitmo — APPROVED** [review]

**Pull request merged**